dockerfile

```diff
@@ -0,0 +1,28 @@
+
+FROM node:16 AS build-stage
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+# Serve the app using Node.js
+FROM node:16 AS production-stage
```

Comment:
Use` node:alpine` to reduce the size of final image

Reply:
Hello @asr2003 I think this are good suggestions!

Reply:
I think it would good to use node latest here as well

Reply:
Yes, we can use node lts for build stage. node-alpine is the lighter image which we can use safely to run the already built files in Stage-1. It's the mantra we follow to reduce docker image :)

Reply:
Cool!